e2e/next/src/next.test.ts

```diff
@@ -184,7 +184,7 @@ describe('Next.js Applications', () => {
 
     runCLI(`generate @nx/next:app ${appName} --no-interactive --style=css`);
 
-    if (runE2ETests('cypress')) {
+    if (runE2ETests('playwright')) {
```

Comment:
Can you rebase this change should be in master IIRC.

Reply:
rebased